app/models/email_template.rb

```diff
@@ -3,26 +3,34 @@ class EmailTemplate < ApplicationRecord
-      new_affiliate_site: "[Search.gov] Your new site: <%= @affiliate.display_name %>",
-      new_affiliate_user: "[Search.gov] You've been added to <%= @affiliate_display_name %>",
-      new_feature_adoption_to_admin: "[Search.gov] Features adopted yesterday",
-      user_email_verification: "[Search.gov] Verify your email",
-      new_user_to_admin: "[Search.gov] New user sign up",
-      password_reset_instructions: "[Search.gov] Reset your password",
-      update_external_tracking_code: "[Search.gov] 3rd Party Tracking",
-      user_sites: '[Search.gov] Searchers now see your Federal Register notices and rules',
-      welcome_to_new_user: "[Search.gov] Welcome to Search.gov",
-      welcome_to_new_user_added_by_affiliate: "[Search.gov] Welcome to Search.gov",
-      user_approval_removed: "[Search.gov] User account set to 'not_approved'",
-      low_query_ctr_watcher: "[Search.gov] {{ctx.metadata.alert_name}} (Custom Alert)",
-      no_results_watcher: "[Search.gov] {{ctx.metadata.alert_name}} (Custom Alert)",
-  }
+    affiliate_header_footer_change: '[Search.gov] Your header and footer '\
+                                    'for <%= @affiliate.display_name %> changed',
+    affiliate_monthly_report:
+      '[Search.gov] Monthly Report for '\
+      '<%= Date::MONTHNAMES[@user_monthly_report.report_date.month.to_i] %>'\
+      ' <%= @user_monthly_report.report_date.year %>',
+    affiliate_yearly_report: '[Search.gov] <%= @report_year %> Year in Review',
+    daily_snapshot:
+      "[Search.gov] Today's Snapshot for <%= @site.name %> on <%= Date.yesterday %>",
+    deep_collection_notification: '[Search.gov] Deep collection created',
+    filtered_popular_terms_report: '[Search.gov] Filtered Popular Terms for Last Week',
+    new_affiliate_site: '[Search.gov] Your new site: <%= @affiliate.display_name %>',
+    new_affiliate_user:
+      "[Search.gov] You've been added to <%= @affiliate_display_name %>",
+    new_feature_adoption_to_admin: '[Search.gov] Features adopted yesterday',
+    user_email_verification: '[Search.gov] Verify your email',
+    new_user_to_admin: '[Search.gov] New user sign up',
+    password_reset_instructions: '[Search.gov] Reset your password',
+    update_external_tracking_code: '[Search.gov] 3rd Party Tracking',
+    user_sites: '[Search.gov] Searchers now see your Federal Register notices and rules',
+    welcome_to_new_user: '[Search.gov] Welcome to Search.gov',
+    welcome_to_new_user_added_by_affiliate: '[Search.gov] Welcome to Search.gov',
+    user_approval_removed: "[Search.gov] User account set to 'not_approved'",
+    low_query_ctr_watcher: '[Search.gov] {{ctx.metadata.alert_name}} (Custom Alert)',
+    no_results_watcher: '[Search.gov] {{ctx.metadata.alert_name}} (Custom Alert)',
+    account_deactivated: '[Search.gov] Your Search.gov account has been deactivated '\
+                         'for the post-deactivation notice'
+  }.freeze
 
```

Comment:
this really should be in a yml file not in here. But for now just cleaned it up a bit.

Reply:
👍